django_6pm/django_6pm/product/models.py

```diff
@@ -1,35 +1,47 @@
 from django.db import models
+from django.utils.translation import gettext_lazy as _
 
 
 class Brand(models.Model):
-    brand_name = models.CharField(max_length=250)
-    brand_logo = models.URLField()
+    name = models.CharField(max_length=250)
+    logo = models.URLField()
 
 
 class Category(models.Model):
-    cat_name = models.CharField(max_length=250)
-    parent_cat_id = models.ForeignKey("self", on_delete=models.SET_NULL, null=True, blank=True)
+    name = models.CharField(max_length=250)
+    parent = models.ForeignKey("self", on_delete=models.SET_NULL, null=True, blank=True)
 
 
 class Product(models.Model):
-    product_id = models.PositiveBigIntegerField(primary_key=True)
-    name = models.CharField(max_length=1000)
-    gender = models.CharField(max_length=100)
+    class Gender(models.IntegerChoices):
+        WOMEN = 1, _("women")
+        MEN = 2, _("men")
+        UNISEX_ADULTS = 3, _("unisex-adults")
+        BOYS = 4, _("boys")
+        GIRLS = 5, _("girls")
+        UNISEX_KIDS = 6, _("unisex-kids")
+
+    name = models.TextField()
+    gender = models.IntegerField(choices=Gender.choices)
     description = models.TextField()
-    currency = models.CharField(max_length=50)
-    brand_id = models.ForeignKey(Brand, on_delete=models.SET_NULL, null=True, blank=True)
-    cat_id = models.ForeignKey(Category, on_delete=models.SET_NULL, null=True, blank=True)
+    currency = models.CharField(max_length=10)
+    brand = models.ForeignKey(Brand, on_delete=models.SET_NULL, null=True, blank=True)
+    category = models.ForeignKey(Category, on_delete=models.SET_NULL, null=True, blank=True)
 
 
-class Sku(models.Model):
-    sku_size = models.CharField(max_length=250)
-    sku_color = models.CharField(max_length=1000)
-    sku_price = models.FloatField()
+class ProdColor(models.Model):
+    color = models.TextField()
```

Comment:
why TextField?

Reply:
Because color names can be really long sometimes

Reply:
Need to fix that color name, color names are not more than 16 chars